dotCMS/src/main/java/com/dotmarketing/listeners/ContextLifecycleListener.java

```diff
@@ -77,8 +77,12 @@ public void contextInitialized(ServletContextEvent arg0) {
 			Logger.error(this,e.getMessage(),e);
 		}
 
-		//Initialises/reconfigures log4j based on a given log4j configuration file
-		Log4jUtil.initializeFromPath(path);
+		// Do not reconfigure if using global configuration.  Remove this if we move
+        // a full global configuration
+        if (System.getProperty("Log4jContextSelector").equals(BasicAsyncLoggerContextSelector.class.getName()))
```

Comment:
Google style please - meaning, needs braces

Reply:
Thought I had the correct formatter on, Fixed.